CONTRIBUTING.md

```diff
@@ -96,3 +96,25 @@ go test -v <path> -run <TestSuite> -testify.m <TestSpercificTaskName>
 go test -v github.com/uber/cadence/common/persistence -run TestCassandraPersistenceSuite -testify.m TestPersistenceStartWorkflow
 ```
 
+## Release Process 
+
+1. Create new version branch in the repo, branch name should be "0.x.y_release" ( we use a branch instead of tag in favor of cherry-picking patches for hot fixes from master)
+2. Run all stress/canary test(for both Cassandra/SQL) and deploy to staging/production. For detailed instructions go to our internal release process wiki)
+3. Go to Release page, click on "Draft a new release" to publish a new release. Follow the format of the previous release for release notes. This will create a tag named "v0.x.x" 
+4. Build the new docker image and push into docker hub.
+```bash
+docker build . -t ubercadence/server:THE.LATEST.VERSION  --build-arg git_branch=vTHE.LATEST.VERSION 
+docker login
+docker push ubercadence/server:THE.LATEST.VERSION
+```
+5. Update "docker-compose.yml" and "docker-compose-mysql.yml" to use latest version and check in to master
+```yaml
+  cadence:
+    image: ubercadence/server:THE.LATEST.VERSION 
+    ports
+```
+6. Create the tar.gz file and upload to release page
+```bash
+cd github.com/uber/cadence/docker
+tar -cvf docker.tar.gz *
+```
```

Comment:
This whole section should be in the internal docs. People contributing to Cadence server won't have permission to push to our Docker or add a release.

Reply:
Yes, that's true. It's also nice to have the release process in a single place rather than split into external and internal. I will move it.